src/ott/solvers/linear/implicit_differentiation.py

```diff
@@ -295,3 +283,42 @@ def gradient(
 
   def replace(self, **kwargs: Any) -> "ImplicitDiff":  # noqa: D102
     return dataclasses.replace(self, **kwargs)
+
+
+def solve_jax_cg(
+    lin: LinOp_t,
+    b: jnp.ndarray,
+    lin_t: Optional[LinOp_t] = None,
+    symmetric: Optional[bool] = False,
+    ridge_identity: float = 0.0,
+    ridge_kernel: float = 0.0,
+    **kwargs: Any
+) -> jnp.ndarray:
+  """Wrapper around JAX native linear solvers.
+
+  Args:
+    lin: Linear operator
+    b: vector such that sought `x` is such that `lin(x)=b`
```

Comment:
Too much "such that", would rephrase as:
Vector :math:`b` such that :math:`A x = b`.

And update the docs of the other function as well.

Reply:
added something a bit less mathematical! (if we want to be math rigorous, it should be `A(x)`)